GitUI/UserControls/FileStatusList.cs

```diff
@@ -826,6 +755,38 @@ private async Task OpenSubmoduleAsync()
             process.Start();
         }
 
+        private void SelectItems(Func<ListViewItem, bool> isToBeSelectedFunc)
+        {
+            try
+            {
+                SuspendLayout();
```

Comment:
I guess, this is pointless, isn't it? (factored out from some of the replaced functions)